GeometryCore/include/Mesh.h

```diff
@@ -8,17 +9,29 @@ namespace geom_utils
 {
     class Mesh
     {
-        std::vector<FPoint3D> vertices;
+        std::vector<Triangle3D> vertices;
+        std::string comment;
```

Comment:
What intention/goal comment has?

Reply:
it contains comment of stl file, I thought it would be useful. Should i remove it?

![image](https://user-images.githubusercontent.com/45259941/82581477-4e0d0880-9b99-11ea-9aa7-6ffc8ae0b7cd.png)

Reply:
I see... then maybe rename this to `std::string modelName`; smth like this

Reply:
ok, i will rename it